app/components/windows/EditStreamInfo.vue.ts

```diff
@@ -81,16 +88,19 @@ export default class EditStreamInfo extends Vue {
 
   encoderProfile: IMultiSelectProfiles;
 
+  facebookPages: { pages: any[], page_id: string };
```

Comment:
Ideally we should define an interface for the pieces of data we use in a page.  It looks like there already might be one defined in the facebook platform service?